src/sources/socket/tcp.rs

```diff
@@ -112,10 +113,10 @@ impl TcpSource for RawTcpSource {
         self.decoder.clone()
     }
 
-    fn handle_events(&self, events: &mut [Event], host: Bytes, byte_size: usize) {
+    fn handle_events(&self, events: &mut [Event], host: Bytes, _byte_size: usize) {
         emit!(&SocketEventsReceived {
```

Comment:
Would it make sense to move this event up to `TcpSource` like `TcpBytesReceived` that way all implementers get it automatically?